web/src/main/java/org/akaza/openclinica/controller/openrosa/OpenRosaSubmissionController.java

```diff
@@ -511,11 +509,10 @@ private boolean mayProceed(Study childStudy, StudySubjectBean ssBean) throws Exc
 
         if (ssBean == null) {
             logger.debug("studyStatus: " + studyStatus);
-            if (studyStatus.equalsIgnoreCase("available"))
+            if (studyStatus.equalsIgnoreCase("available") || studyStatus.equalsIgnoreCase("frozen"))
```

Comment:
I know it is existing code but since we're adding to it, is it possible to use the Status constants? I believe they're defined.